web/src/lib/components/forms/library-rename-form.svelte

```diff
@@ -22,14 +21,18 @@
 </script>
 
 <form {onsubmit} autocomplete="off">
-  <FullScreenModal icon={mdiRenameOutline} title={$t('rename')} onClose={onCancel}>
-    <Field label={$t('name')}>
-      <Input bind:value={newName} />
-    </Field>
+  <Modal icon={mdiRenameOutline} title={$t('rename')} onClose={onCancel} size="small">
+    <ModalBody>
+      <Field label={$t('name')}>
+        <Input bind:value={newName} />
+      </Field>
+    </ModalBody>
 
-    {#snippet stickyBottom()}
-      <Button shape="round" fullWidth color="secondary" onclick={onCancel}>{$t('cancel')}</Button>
-      <Button shape="round" fullWidth type="submit">{$t('save')}</Button>
-    {/snippet}
-  </FullScreenModal>
+    <ModalFooter>
+      <div class="flex gap-2 w-full">
+        <Button shape="round" fullWidth color="secondary" onclick={onCancel}>{$t('cancel')}</Button>
+        <Button shape="round" fullWidth type="submit">{$t('save')}</Button>
+      </div>
+    </ModalFooter>
+  </Modal>
```

Comment:
This won't work, as the content of the Modal is Portal'd all the way up to the body, outside the form. You need the form to be inside the `ModalBody`, give the form a descriptive `id`, and then reference that id in the submit button with `form="<id>"`

Reply:
Good to know!